sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -128,6 +128,7 @@
 
   /* Global settings */
   String SONAR_HOME = "SONAR_HOME";
+  @Deprecated
   String PROJECT_BRANCH_PROPERTY = "sonar.branch";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)